docs/pipeline/tutorials/metadata.ts

```diff
@@ -25,7 +25,9 @@ export async function generateMetadata(
     allFiles: Object.keys(files),
     tutorialFiles,
     answerFiles: await getAnswerFiles(path, config, files),
-    hiddenFiles: [],
+    hiddenFiles: config.openFiles
+      ? Object.keys(tutorialFiles).filter((filename) => !config.openFiles!.includes(filename))
```

Comment:
Since `openFiles` could be undefined, we should just use optional chaining here

```suggestion
      ? Object.keys(tutorialFiles).filter((filename) => !config.openFiles?.includes(filename))
```

Reply:
It can't be undefined since we null check it in the ternary one level up.

Reply:
Does typescript not reconcile that for us then?  I can merge as is then.